packages/navigation/package.json

```diff
@@ -39,17 +39,18 @@
     "@launchpad-ui/popover": "workspace:~",
     "@launchpad-ui/tokens": "workspace:~",
     "@launchpad-ui/tooltip": "workspace:~",
-    "@react-aria/utils": "3.14.1",
-    "@react-stately/list": "3.6.0",
-    "@react-types/shared": "3.16.0",
+    "@react-aria/utils": "3.15.0",
```

Comment:
We are actually using some utils/types throughout LaunchPad that aren't exported from `react-aria` or `react-stately` monorepo packages. I put up [a PR with react-spectrum](https://github.com/adobe/react-spectrum/pull/4182) to see if they're receptive to additional exports. There may be some reason these _aren't_ exported.